### PR TITLE
Add meeting examplecontent.

### PR DESCRIPTION
**Pull request by deiferni**
This PR adds initial meeting example content to our development profile. It also adds a reusable municipality example content profile which can be used in our demo installations.

It also fixes:
- Creating sablon templates with our customized inflator pipeline should now correctly create initial versions.
- Change how an agenda-item is appended to meetings after creation to make it visible for meetings directly without flushing the session.

### CODE REVIEW

**Comment by deiferni** [comment]
@phgross plz have a look

File: opengever/testing/builders/activity.py [hunk 1]
```diff
@@ -5,7 +5,12 @@
 from opengever.activity.model import Watcher
 from opengever.base.oguid import Oguid
 from opengever.ogds.models.tests.builders import SqlObjectBuilder
-from plone.app.testing import TEST_USER_ID
+
+
+try:
```
Comment (phgross) [inline opengever/testing/builders/activity.py]: Gäbe es hier evtl. eine Alternative um dieses Problem zu umgehen? 

Falls nicht wäre gut wenn du zumindest die Überprüfung folgendermassen implementieren würdest. 

```python
try:
    pkg_resources.get_distribution('ftw.solr')
    from ftw.solr.interfaces import IFtwSolrLayer
except pkg_resources.DistributionNotFound:
    HAS_FTW_SOLR = False
else:
    HAS_FTW_SOLR = True
```

Comment (deiferni) [inline opengever/testing/builders/activity.py]: sure. done :wink:.

File: opengever/examplecontent/hooks.py [hunk 2]
```diff
@@ -10,29 +14,165 @@
...
+    def setup_builders(self):
+        session.current_session = session.BuilderSession()
+        session.current_session.session = self.db_session
+
+    def create_content(self):
+        self.create_members()
+        self.create_meetings()
+        self.create_proposals()
+
+    def create_members(self):
```
Comment (phgross) [inline opengever/examplecontent/hooks.py]: create_members_and_memberships ...

File: opengever/examplecontent/hooks.py [hunk 3]
```diff
@@ -10,29 +14,165 @@
...
+                                     time(12, 0))
+                )
+            )
+
+    def create_proposals(self):
+        proposal1 = create(
+            Builder('proposal')
+            .within(self.dossier_baumann)
+            .having(committee=self.committee_assembly_model,
+                    title=u'Genehmigung der Anstellung von Hans Baumann als '
```
Comment (phgross) [inline opengever/examplecontent/hooks.py]: Ich denke es wäre übersichtlicher wenn wir alle Proposals gleich erstellen würden. Also nicht nur teilweise der Inhalt in eine Konstante verschieben ....

**Comment by phgross** [comment]
looks good, just my comments

File: opengever/examplecontent/hooks.py (hunk 3 above)
Comment (deiferni) [inline opengever/examplecontent/hooks.py]: :+1:

**Comment by lukasgraf** [comment]
@deiferni this PR not only *adds* meeting example content / new repositories / new profiles, it also *changed* the default setup. Please mention changes like this in the PR.

Also, the first three dossiers in `municipality_content/opengever_content/01_initial_dossiers.json` can't be created because their containers don't exist:

```
2015-04-24 16:57:23 WARNING collective.transmogrifier.constructor Container ordnungssystem/fuehrung/kommunikation/allgemeines does not exist for item /ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1
2015-04-24 16:57:23 WARNING collective.transmogrifier.constructor Container ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1 does not exist for item /ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/dossier-2
2015-04-24 16:57:23 WARNING collective.transmogrifier.constructor Container ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1 does not exist for item /ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/document-1
2015-04-24 16:57:23 WARNING collective.transmogrifier.constructor Container ordnungssystem/fuehrung/interne-organisation/planung-und-organisatorisches does not exist for item /ordnungssystem/fuehrung/interne-organisation/planung-und-organisatorisches/dossier-2
2015-04-24 16:57:23 WARNING collective.transmogrifier.constructor Container ordnungssystem/fuehrung/interne-organisation/planung-und-organisatorisches does not exist for item /ordnungssystem/fuehrung/interne-organisation/planung-und-organisatorisches/dossier-3
```

**Pull request merged**